src/components/Kv/KvFlag2.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+	<div :class="`kv-flag kv-flag--${aspectRatio}`">
+		<div class="kv-flag__wrapper">
+			<template v-if="fromSprite">
+				<div
+					class="kv-flag__img kv-flag__img--sprite"
+					:style="{ backgroundPositionY: spriteYPosition + '%' }"
+				></div>
+			</template>
+			<template v-else>
+				<component
+					:is="flagSVG"
+					class="kv-flag__img"
+				/>
+			</template>
+			<span class="show-for-sr">{{ countryName }}</span>
+		</div>
+	</div>
+</template>
+
+<script>
+import { getCodes, getName } from 'country-list';
```

Comment:
Already noted this in the other PR, but I'll mention it here as well; the country-list package has slightly different country codes than we use, and it's also missing Kosovo, so we should account for that or use something else.